enforcer-parent/enforcer/src/main/resources/Dockerfile

```diff
@@ -66,7 +66,7 @@ RUN chmod +x ${GRPC_HEALTH_PROBE_PATH}
 WORKDIR ${MG_USER_HOME}
 USER ${MG_USER}
 
-COPY maven/ lib/
+COPY maven/lib/ lib/
```

Comment:
Shall we do the same for Dockerfile.ubuntu

Reply:
missed it. added now